### PR TITLE
feat: add more type to DynamicField & add popup type to DynamicLayout

### PR DESCRIPTION
**Pull request by WANZARGEN**
## To Reviewers
- [ ] Skipping reviews is not a problem

## Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### `more` type is added to DynamicField
image
image

### `displayKey` is added to DynamicField typeOptions
image

### `popup` type is added to DynamicLayout
image

### `popupVisible` is added to typeOptions and `update-popup-visible` event is added in DynamicLayout
image
스크린샷 2022-07-05 오후 4 51 29

### CODE REVIEW

File: src/data-display/dynamic/dynamic-field/templates/more/index.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    computed,
+    defineComponent, PropType, reactive, toRefs,
+} from '@vue/composition-api';
+
+import { MoreDynamicFieldProps, MoreTypeOptions } from '@/data-display/dynamic/dynamic-field/templates/more/type';
+import PDynamicLayout from '@/data-display/dynamic/dynamic-layout/PDynamicLayout.vue';
+import { getValueByPath } from '@/data-display/dynamic/helper';
+import PTextList from '@/others/console/text-list/PTextList.vue';
+
+const supportedTypes = ['popup'];
```
Comment (yuda110) [inline src/data-display/dynamic/dynamic-field/templates/more/index.vue:35]: how about `SUPPORTED_TYPES`?

**Review by yuda110 — COMMENTED** [review]
LGTM!

File: src/data-display/dynamic/dynamic-layout/templates/popup/index.vue [hunk 2]
```diff
@@ -56,14 +56,26 @@ export default defineComponent<PopupDynamicLayoutProps>({
...
+        });
+
+        const handleUpdateVisible = (popupVisible) => {
+            state.popupVisible = popupVisible;
+            emit('update-popup-visible', popupVisible);
+        };
+        watch(() => props.typeOptions, (typeOptions) => {
+            if (typeOptions?.popupVisible !== state.popupVisible) {
+                state.popupVisible = typeOptions?.popupVisible;
+                // emit('update-popup-visible', state.popupVisible);
```
Comment (JeoungSulMo) [inline src/data-display/dynamic/dynamic-layout/templates/popup/index.vue:72]: I think you can remove it!

Comment (WANZARGEN) [reply src/data-display/dynamic/dynamic-layout/templates/popup/index.vue:72]: thx :)

Comment (WANZARGEN) [reply src/data-display/dynamic/dynamic-layout/templates/popup/index.vue:72]: oh you mean watch effect?

**Review by JeoungSulMo — APPROVED** [review]
LGTM!!

File: src/data-display/dynamic/dynamic-layout/templates/popup/index.vue (hunk 2 above)
Comment (yuda110) [reply src/data-display/dynamic/dynamic-layout/templates/popup/index.vue]: I think what he's talking about is a comment 😅

**Pull request merged**